cohortextractor/definition/__init__.py

```diff
@@ -0,0 +1,8 @@
+from .definition import Cohort, pick_first_value, register
```

Comment:
I'm assuming that we'll have more of these definition things later and will probably want to split them out in some sensible way, but for now they're all in one `definition.py` file